stream-chat-android-ui-components/src/main/res/values-es/strings_message_composer.xml

```diff
@@ -14,26 +14,15 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 -->
-<resources>
-    <string name="stream_ui_message_composer_send_to_channel">"También enviar al canal"</string>
-    <string name="stream_ui_message_composer_mode_reply">"Contestar"</string>
-    <string name="stream_ui_message_composer_mode_edit">"Editar Mensaje"</string>
-    <string name="stream_ui_message_composer_hint_normal">"Escribe algo aquí"</string>
-    <string name="stream_ui_message_composer_hint_cannot_send_message">You can\'t send messages in this channel</string>
-
-    <!-- Commands Popup -->
-    <string name="stream_ui_message_composer_instant_commands">"Comandos instantáneos"</string>
-
-    <!-- File Picker Dialog -->
-    <string name="stream_ui_message_composer_gallery_access">"Permitir acceso a tu galería"</string>
-    <string name="stream_ui_message_composer_files_access">"Permitir acceso a tus ficheros"</string>
-    <string name="stream_ui_message_composer_camera_access">"Permitir acceso a la cámara"</string>
-    <string name="stream_ui_message_composer_recent_files">"Ficheros recientes"</string>
-    <string name="stream_ui_message_composer_no_files">"No hay fotos ni vídeos"</string>
-
-    <!-- Validation Errors -->
-    <string name="stream_ui_message_composer_error_message_length">Tamaño máximo de mensaje (%d) excedido.</string>
```

Comment:
🤔 What happened with these strings?

Reply:
It is weird
Let me check Localazy translations